### PR TITLE
sample(): added sample for mirkoorm app with terminus

### PR DESCRIPTION
**Pull request by Tony133**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/terminus/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
Proposal:
- Added a sample for using terminus  with mirkoorm

P.s. Terminus ♥️

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

## Other information
I hope it can be useful to people using terminus with mirkoorm

### CODE REVIEW

File: sample/011-mirkoorm-app/package.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
+{
+  "name": "011-mirkoorm-app",
+  "version": "0.0.1",
+  "private": true,
+  "license": "UNLICENSED",
+  "scripts": {
+    "build": "nest build",
+    "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
+    "start": "nest start",
```
Comment (BrunnerLivio) [inline sample/011-mirkoorm-app/package.json:9]:
```suggestion
    "start": "ts-node src/main.ts",
```

File: sample/011-mirkoorm-app/package.json [hunk 2]
```diff
@@ -0,0 +1,69 @@
+{
+  "name": "011-mirkoorm-app",
+  "version": "0.0.1",
+  "private": true,
+  "license": "UNLICENSED",
```
Comment (Tony133) [inline sample/011-mirkoorm-app/package.json:5]:
```suggestion
  "license": "MIT",
```

**Comment by Tony133** [comment]
Very strange for the CI to fail, @BrunnerLivio  do you have any ideas?

Screenshot 2023-02-07 alle 21 49 47

File: sample/011-mirkoorm-app/package.json [hunk 3]
```diff
@@ -0,0 +1,69 @@
+{
+  "name": "011-mirkoorm-app",
+  "version": "0.0.1",
+  "private": true,
+  "license": "MIT",
+  "scripts": {
+    "build": "nest build",
```
Comment (BrunnerLivio) [inline sample/011-mirkoorm-app/package.json:7]:
```suggestion
    "build": "rm -rf dist && tsc -p tsconfig.json",
```

**Comment by BrunnerLivio** [comment]
Thanks @Tony133 appreciated as always <3

**Pull request merged**